debug/8583.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
```

Comment:
Are you planning to commit this file? It would be more useful for the next developer if it was named more appropriately.